config/repo-values.yaml

```diff
@@ -17,4 +17,6 @@ sync:
     - name: buildkite
       url: https://buildkite.github.io/charts
     - name: keel
-      url: https://charts.keel.sh 
+      url: https://charts.keel.sh
+    - name: appscode
+      url: https://charts.appscode.com/stable/
```

Comment:
Hi @tamalsaha The `/stable` path in this URL doesn't resolve. However without that path, it does:
```console
$ curl -s -o /dev/null -I -w "%{http_code}" https://charts.appscode.com/stable/
404
$ curl -s -o /dev/null -I -w "%{http_code}" https://charts.appscode.com
200
```